webrtc-stats.html

```diff
@@ -576,6 +558,10 @@ <h3>
               ICE candidate pair statistics related to the <code>RTCIceTransport</code> objects. It
               is accessed by the <code><a>RTCIceCandidatePairStats</a></code>.
             </p>
+            <p>
+              A candidate pair is deleted when the RTCIceTransport does an ICE restart, and the
+              candidate pair is no longer in use.
```

Comment:
When exactly is that? Is it when the [statechange](http://w3c.github.io/webrtc-pc/#event-icetransport-statechange) event is fired? Is that from SLD?

If so, should we say it MUST fire before SLD's promise is resolved?

Reply:
Or is it right after that ICE changes happen?

Reply:
I think it's when the ICE transport changes state to "new".